src/scripts/modules/media/body/body.coffee

```diff
@@ -157,7 +157,7 @@ define (require) ->
           .wrapInner('<section class="ui-body">')
           .prepend('''
             <div class="ui-toggle-wrapper">
-              <button class="btn-link ui-toggle" title="Show/Hide Solution"></button>
+              <button class="btn-link ui-toggle" title="Show/Hide Solution" aria-label="show/hide solution"></button>
```

Comment:
How does this convey to the screen reader that this section is expandable or that the text changes when toggled. The aria-label is a good idea, but the solution should be in the CSS where the text on the button is changed.